lisa/features/infiniband.py

```diff
@@ -424,7 +437,7 @@ def install_ofed(self) -> None:
         node.execute(
             "/etc/init.d/openibd force-restart",
             expected_exit_code=0,
-            expected_exit_code_failure_message="SetupRDMA: failed to restart driver",
+            expected_exit_code_failure_message="SetupRDMA: failed to " "restart driver",
```

Comment:
```expected_exit_code_failure_message="SetupRDMA: failed to restart driver",```

Reply:
An error caused by black. Have fixed it.